ltac/tacinterp.ml

```diff
@@ -1145,11 +1146,8 @@ let warn_deprecated_info =
             strbrk "There is an \"Info\" command to replace it." ++fnl () ++
             strbrk "Some specific verbose tactics may also exist, such as info_eauto.")
 
-let rec arg_names vars =
-  match vars with
-  | [] -> Pp.str ""
-  | None::vars' -> Pp.str "_ " ++ arg_names vars'
-  | Some v :: vars' -> pr_id v ++ Pp.str " " ++ arg_names vars'
+let rec arg_names (vars: Names.Id.t option list) =
+  Pp.prlist_with_sep spc (fun v -> Option.default (Pp.str "_") (Option.map pr_id v)) vars
```

Comment:
I wonder if maybe nameops would be a better place for this, so it shared among all callers.